userapi/src/main/java/fk/prof/userapi/Configuration.java

```diff
@@ -0,0 +1,107 @@
+package fk.prof.userapi;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.json.JsonObject;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.util.Map;
+
+/**
+ * Created by gaurav.ashok on 15/05/17.
+ */
+public class Configuration {
+
+  @JsonProperty("profile.retention.duration.min")
+  public Integer profileRetentionDurationMin = 30;
```

Comment:
I have kept them public deliberatly because this way it works with Jackson without any setters and if there is any configuration change then there is less headache to manage there getters/setters.

Also since all jsonObject previously were mutable anyway. This class is for our use and we can make sure that we do not update it later.